src/assets/main.css

```diff
@@ -45,6 +45,84 @@ For example over-ride the colour of the btn-contained backgound using bg-scarlet
   .btn-icon {
     @apply fill-current h-5 w-5;
   }
+
+  /* Button base class */
+  .btn {
+    @apply rounded-3xl inline-flex justify-center items-center space-x-2;
+  }
+
+  /* Button base class for icon only buttons */
+  .btn-icon-base {
+    @apply rounded-full inline-flex justify-center items-center;
+  }
+
+  /* Button styles */
```

Comment:
Note that these colours will be changed to forest green rather than blumine when we change all the site colours over